src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/Example.rst

```diff
@@ -0,0 +1,256 @@
+
+This benchmark example uses no external input files and everything required is
+contained within two GEOSX xml files that are located at:
+
+.. code-block:: console
+
+  inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+
+and
+
+.. code-block:: console
+
+  inputFiles/wellbore/CasedThermoElasticWellbore_benchmark.xml
+
+The corresponding integrated test is
+
+.. code-block:: console
+
+  inputFiles/wellbore/CasedThermoElasticWellbore_smoke.xml
+
+-----------------------------------------------------------
+Geometry and Mesh
+-----------------------------------------------------------
+
+The internal wellbore mesh generator ``InternalWellbore`` is employed to create the mesh of this wellbore problem. The radii of the casing cylinder, the cement sheath cylinder and the far-field boundary of the surrounding rock formation are defined by a vector ``radius``. In the tangent direction, ``theta`` angle is specified from 0 to 90 degree for a quarter of the geometry regarding the axisymmetry of the problem. The problem is plane strain and radial thermal diffusion on therefore only one single element in z-axis is needed. The trajectory of the well is defined by ``trajectory``, which is vertical in this case. The ``autoSpaceRadialElems`` parameters allow optimally increasing the element size from local zone around the wellbore to the far-field zone. In this example, the auto spacing option is only applied for the rock formation. The ``useCartesianOuterBoundary`` with a value 3 specified for the rock layer transforms the far-field boundary to a circular shape. The ``cellBlockNames`` and ``elementTypes`` define the regions and related element types associated to casing, cement sheath and rock. 
+ 
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_WellboreMesh -->
+  :end-before: <!-- SPHINX_WellboreMeshEnd -->
+
+.. _meshCasedThermalElasticWellboreFig:
+.. figure:: mesh.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   An optimized mesh for the considering cased wellbore.
+
+-----------------------------------------------------------
+Material properties
+-----------------------------------------------------------
+
+The bulk and shear drained elastic moduli of the materials as well as its drained linear thermal expansion coefficient relating temperature change to stress and displacement changes are defined within the ``constitution`` tag as follows:
+ 
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_ThermoElasticProperties -->
+  :end-before: <!-- SPHINX_ThermoElasticPropertiesEnd -->
+
+Here the solid density is also defined but it is not used because the gravitational effect is ignored in this example. To mimic a thermoelastic coupling without fluid flow, a negligible porosity and a zero Biot's coefficient are defined as:
```

Comment:
```suggestion
Here the solid density is also defined but it is not used because the gravitational effect is ignored in this example. To mimic a thermoelastic coupling without fluid flow, a negligible porosity and a zero Biot coefficient are defined as:
```

Reply:
done